pkg/operator/staticpod/controller/guard/guard_controller.go

```diff
@@ -77,6 +79,7 @@ func NewGuardController(
 		podResourcePrefix:       podResourcePrefix,
 		operatorName:            operatorName,
 		readyzPort:              readyzPort,
+		readyzEndpoint:          readyzEndpoint,
```

Comment:
this lgtm.  Is it stuck on something in particular?

Reply:
Just preempted by higher priority tasks